src/nvim/tui/tui.c

```diff
@@ -350,6 +351,10 @@ static void tui_resize(UI *ui, int width, int height)
   data->scroll_region.left = 0;
   data->scroll_region.right = width - 1;
   data->row = data->col = 0;
+  // try to resize the terminal window
+  char r[16];  // enough for 9999x9999
+  snprintf(r, sizeof(r), "\x1b[8;%d;%dt", height, width);
```

Comment:
@fmoralesc If I'm reading http://invisible-island.net/xterm/ctlseqs/ctlseqs.html correctly it seems we should use `4` instead of `8`. Do you recall why you choose `8`?

```
CSI Ps ; Ps ; Ps t
          Window manipulation (from dtterm, as well as extensions).
          ...
            Ps = 4  ;  height ;  width -> Resize the xterm window to
          given height and width in pixels.  Omitted parameters reuse
          the current height or width.  Zero parameters use the dis-
          play's height or width.
            ...
            Ps = 8  ;  height ;  width -> Resize the text area to given
          height and width in characters.  Omitted parameters reuse the
          current height or width.  Zero parameters use the display's
          height or width.
```